lib/puppet/provider/cron/crontab.rb

```diff
@@ -1,3 +1,4 @@
+require 'puppet/provider/cron/filetype'
```

Comment:
Need to use `relative_require` here (or pass a relative path to `require`) so that the helper code can be loaded when compiling a catalog on the server (since the environment specific module path is not added to `$LOAD_PATH`). This isn't an issue for windows exe/msi because those are in core puppet.

I asked @tphoney about creating a puppet-lint rule to try to detect this.